File: UIKitCommonLocalization/ViewController.m
//
//  ViewController.m
//  UIKitCommonLocalization
//
//  Created by Maximilian Christ on 05/07/14.
//  Copyright (c) 2014 McZonk. All rights reserved.
//

#import "ViewController.h"

#import "NSBundle+CommonLocalizations.h"
#import "NSBundle+UIKitBundle.h"

@implementation ViewController
            
- (void)viewDidLoad {
	[super viewDidLoad];
	
	UIBarButtonItem *button = [[UIBarButtonItem alloc] init];
	button.title = NSBundle.UIKitBundle.localizedDone;
	self.navigationItem.rightBarButtonItem = button;
}

@end

File: UIKitCommonLocalizationSwift/ViewController.swift
//
//  ViewController.swift
//  UIKitCommonLocalizationsSwift
//
//  Created by Maximilian Christ on 05/07/14.
//  Copyright (c) 2014 McZonk. All rights reserved.
//

import UIKit


class ViewController: UIViewController {
                            
	override func viewDidLoad() {
		super.viewDidLoad()
		
		let bundle = NSBundle.UIKitBundle()
		
		let button = UIBarButtonItem()
		button.title = bundle.localizedDone()
		navigationItem.rightBarButtonItem = button;
	}

}
